src/tests/ftest/aggregation/multiple_pool_cont.py

```diff
@@ -60,11 +67,11 @@ def test_free_space(self, space_tag1, space_tag2):
                                  self.free_space_dict[pool.uuid][space_tag2])
             self.log.info("Pool %s Space used by ior = %s", pool.uuid,
                           space_used_by_ior)
-            self.assertTrue(self.free_space_dict[pool.uuid][space_tag1] >
-                            self.free_space_dict[pool.uuid][space_tag2],
-                            "IOR run was not successful.")
+            self.assertGreater(self.free_space_dict[pool.uuid][space_tag1],
+                               self.free_space_dict[pool.uuid][space_tag2],
+                               "IOR run was not successful.")
```

Comment:
Ideally, I think the fail message should be more descriptive. Maybe something like this?
"Free space did not decrease after IOR"

Otherwise, when looking at failed tests, the logs have to be looked through to figure out why the test failed.

Reply:
If I need to fix some other issue later, will fix it.

Reply:
Done.